iOS10-HW14-AntonIsakov/Sourses/Model/CompositionalItem.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+struct CompositionalItem: Hashable {
+    var description: String?
+    var image: UIImage?
+    var numberOfPhotos: Int?
+    
+}
+
+struct CompositionalSection: Hashable {
+    var name: String
+    var items: [CompositionalItem]
+}
+
+extension CompositionalSection {
+    
+    static var modelSectionArray = [
+        CompositionalSection(name: "Мои альбомы",
+                             items: [CompositionalItem(description: "Недавние", image: UIImage(named: "BMW") ?? UIImage() , numberOfPhotos: 6),
+                                     CompositionalItem(description: "Избранное", image: UIImage(named: "tiger") ?? UIImage(), numberOfPhotos: 226),
+                                     CompositionalItem(description: "WhatsApp", image: UIImage(named: "WhiteBear") ?? UIImage(), numberOfPhotos: 436),
+                                     CompositionalItem(description: "Instagram", image: UIImage(named: "moto") ?? UIImage(), numberOfPhotos: 116),
+                                     CompositionalItem(description: "Viber", image: UIImage(named: "Sea") ?? UIImage(), numberOfPhotos: 73),
+                                     CompositionalItem(description: "Another", image: UIImage(named: "dog") ?? UIImage(), numberOfPhotos: 56),
+                                     CompositionalItem(description: "Facebook", image: UIImage(named: "dog2") ?? UIImage(), numberOfPhotos: 54),
+                                     CompositionalItem(description: "My photo", image: UIImage(named: "bob") ?? UIImage(), numberOfPhotos: 21)
+                                    ]),
```

Comment:
```suggestion
        CompositionalSection(
            name: "Мои альбомы",
            items: [
                CompositionalItem(description: "Недавние", image: UIImage(named: "BMW"), numberOfPhotos: 6),
                CompositionalItem(description: "Избранное", image: UIImage(named: "tiger"), numberOfPhotos: 226),
                CompositionalItem(description: "WhatsApp", image: UIImage(named: "WhiteBear"), numberOfPhotos: 436),
                CompositionalItem(description: "Instagram", image: UIImage(named: "moto"), numberOfPhotos: 116),
                CompositionalItem(description: "Viber", image: UIImage(named: "Sea"), numberOfPhotos: 73),
                CompositionalItem(description: "Another", image: UIImage(named: "dog"), numberOfPhotos: 56),
                CompositionalItem(description: "Facebook", image: UIImage(named: "dog2"), numberOfPhotos: 54),
                CompositionalItem(description: "My photo", image: UIImage(named: "bob"), numberOfPhotos: 21)
            ]
        ),
```

ниже по аналогии с этой секцией исправляй